git_machete/cmd.py

```diff
@@ -3117,7 +3182,7 @@ def usage(c: str = None) -> None:
               <b>-f, --fork-point=<fork-point-commit></b>    Specifies the alternative fork point commit after which the squashed part of history is meant to start.
         """,
         "status": """
-            <b>Usage: git machete s[tatus] [--color=WHEN] [-l|--list-commits] [-L|--list-commits-with-hashes]</b>
+            <b>Usage: git machete s[tatus] [--color=WHEN] [-l|--list-commits] [-L|--list-commits-with-hashes] [--merge]</b>
```

Comment:
```suggestion
            <b>Usage: git machete s[tatus] [--color=WHEN] [-l|--list-commits] [-L|--list-commits-with-hashes] [-M|--merge]</b>
```

but see my comment about alternative naming for this option.